modules/storage/storage.go

```diff
@@ -71,3 +97,31 @@ func Init() error {
 
 	return nil
 }
+
+func initLFS() error {
+	var err error
+	switch setting.LFS.StoreType {
+	case "local":
+		LFS, err = NewLocalStorage(setting.LFS.ContentPath)
+	case "minio":
+		minio := setting.LFS.Minio
+		LFS, err = NewMinioStorage(
+			context.Background(),
+			minio.Endpoint,
+			minio.AccessKeyID,
+			minio.SecretAccessKey,
+			minio.Bucket,
+			minio.Location,
+			minio.BasePath,
+			minio.UseSSL,
+		)
+	default:
+		return fmt.Errorf("Unsupported LFS store type: %s", setting.LFS.StoreType)
+	}
```

Comment:
Is there really no way to make these self-register?

Reply:
I don't know. But the current codes look clear and simple.